### PR TITLE
Ubuntu/Debian Release

### PR DESCRIPTION
**Pull request by souldanger**
updated marquee
added news post
various other minor corrections

### CODE REVIEW

File: _marquee/marquee.md [hunk 1]
```diff
@@ -4,4 +4,4 @@ title: "Marquee"
 ---
 
 
-[</> **Now available** Version 2017.10 **for Windows, macOS, openSUSE Leap 42.2, 42.3, Tumbleweed, Fedora 25 + 26 as well as RHEL/Centos 7** - go to **Download section** </>](/downloads/ "download 2017.10 for Windows,Mac OSX, openSUSE, Fedora and RHEL/Centos")
+[</> **Version 2017.10** now available for **Windows, macOS, openSUSE Leap/Tumbleweed, Fedora, RHEL/CentOS** - read the latest **NEWS** </>](/#news "read the latest News")
```
Comment (jengelh) [inline _marquee/marquee.md]: "news", not "News".

File: downloads/linux.md [hunk 2]
```diff
@@ -33,7 +33,8 @@ current version 58.0
 <div class="dlinux fl-opensuse"></div>
 <header>
 ### Repositories for openSUSE #
-current version 2017.10
+current version 2017-10
```
Comment (jengelh) [inline downloads/linux.md]: No no, it's "2017.10", not "2017-10".

File: _posts/2017-10-21-win-mac-release-v61.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 ---
 layout: post
 style: style2
-title:  "Version 2017.10 now available for Windows and macOS"
-description: Iridium Browser version 2017.10 builds for Windows and macOS now available! Releases for the Debian/Ubuntu, openSUSE and Fedora to follow soon.
+title:  "Version 2017.10 now available for Windows and mac OS"
```
Comment (jengelh) [inline _posts/2017-10-21-win-mac-release-v61.md]: Inconsistent spelling of mac( )OS introduced.

File: downloads/windows.md [hunk 4]
```diff
@@ -37,7 +39,9 @@ current version 2017-10
 [64-bit Download](https://downloads.iridiumbrowser.de/windows/iridiumbrowser-latest-x64.zip "download Portable 64-bit Version"){:.button .small .fit .download1 .icon .fa-download}
 </div>
 **Please note!**        
-Running Iridium portable version works out of the box. As a recommendation we recommend to create a shortcut using the “--user-data-dir=/IridiumUserData” configuration option to store the user data to the preferred location (like a USB stick, etc.)
+<strong>* </strong>= Running Iridium portable version works out of the box. As a recommendation we recommend to create a shortcut using the
```
Comment (jengelh) [inline downloads/windows.md]: "as a recommendation we recommend to", uh, why not just write "We recommend to".

File: _footer/footer.md [hunk 5]
```diff
@@ -9,7 +9,7 @@ Please report any issues and address any questions to our <strong><a href="https
 <br/>
 For more development information please <strong><a href="/development.html" title="more development information">check here</a></strong>
 <br/>
-<span class="fa fa-code"> </span> download the <strong><a href="/downloads/source.html" title="download the Source Code">Source Code</a></strong>
+<span class="fa fa-code"> </span>  download the <strong><a href="/downloads/source.html" title="download the Source Code">Source Code</a></strong>
```
Comment (jengelh) [inline _footer/footer.md]: and what strange Markdown feature is enabled by _this_ extra space?

File: _posts/2017-10-31-linux-release-2017.10.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
+---
+layout: post
+style: style2
+title:  "Latest Debian / Mint / Ubuntu Release Now Available"
+description: We have changed the version numbers and future builds will have the format YYYY.MM
```
Comment (jengelh) [inline _posts/2017-10-31-linux-release-2017.10.md]: The news post's content/description has nothing to do with the title.

Comment (souldanger) [reply _posts/2017-10-31-linux-release-2017.10.md]: Thanks for spotting this - fixed!

**Pull request merged**